runtime/compiler/p/env/J9CPU.cpp

```diff
@@ -26,84 +26,92 @@
 #include "j9.h"
 #include "j9port.h"
 
-
+// supportsFeature and supports_feature_test will be removed when old_apis are no longer needed
```

Comment:
Just trying to understand: why would supportsFeature be removed if you introduced calls to it in this PR?

Also, if supports_feature_test is only used within this class, can we make it private?

Reply:
The end goal is for every single platform to use this `bool OMR::CPU::supportsFeature(uint32_t feature)` API.
Currently we want to make sure that the new APIs are getting called correctly by comparing their query results to the old APIs. I added the tests in `bool J9::CPU::Power::supportsFeature()` and `bool J9::CPU::Power::isAtLeast()`. Once we no longer need the test then these overrides can be removed and we will fall back to use the `bool OMR::CPU::supportsFeature(uint32_t feature)` API in the end.

Yes I can make supports_feature_test private.